src/JuMP.jl

```diff
@@ -346,6 +346,10 @@ function setValue(v::Variable, val::Number)
     end
 end
 
+setValue(set::Array{Variable}, val::Array) = map(setValue, set, val)
+
+setValue(set::JuMPArray{Variable}, val::Array) = setValue(set.innerArray, val)
```

Comment:
Your example gives me the error
```julia
ERROR: `setValue` has no method matching setValue(::JuMPDict{Variable,2}, ::Array{Int64,1})
```
in v0.3.12. Maybe there have been some changes in v0.4?